client/src/components/molecules/MyDailyTimeRecordTable/columns.tsx

```diff
@@ -20,6 +20,7 @@ import WorkStatusChip from '~/components/atoms/WorkStatusChip'
 import { NO_OVERTIME } from '~/utils/constants/overtimeStatus'
 import ChangeShiftRequestModal from './ChangeShiftRequestModal'
 import RequestEditEntryModal from './RequestEditEntryModal'
+import { hasProofFromQuery } from '../../../utils/timeEntryUtils'
```

Comment:
For this one, to minimize the changes, just give alias when importing the helper

```suggestion
import { hasProof as hasProofHelper } from '../../../utils/timeEntryUtils'
```